### PR TITLE
Make test compatible with PHP 8

### PR DESCRIPTION
**Pull request by carusogabriel**

### CODE REVIEW

**Comment by carusogabriel** [comment]
Looks like this test can't be compatible with both PHP 7 and 8.

@kocsismate Looks like these renamings are gonna be a HUGE BC for libraries' tests.

We can think about an `--EXPECTF-8--` or something for `run-tests.php`

File: tests/yaml_parse_file_002.phpt [hunk 1]
```diff
@@ -22,6 +22,6 @@ Warning: yaml_parse_file(): Filename cannot be empty in %s on line %d
 
 Warning: yaml_parse_file(): Filename cannot be empty in %s on line %d
 
-Warning: yaml_parse_file(invalid): failed to open stream: No such file or directory in %s on line %d
+Warning: yaml_parse_file(invalid): %sailed to open stream: No such file or directory in %s on line %d
```
Comment (cmb69) [inline tests/yaml_parse_file_002.phpt]: A possible alternative could be checking for an arbitrary single character (`%c`), or for a regex character class (`%r[Ff]%r`).

Comment (carusogabriel) [reply tests/yaml_parse_file_002.phpt]: @cmb69 Thanks, I didn't know about `%r` :)

Comment (cmb69) [reply tests/yaml_parse_file_002.phpt]: All supported tags are documented at http://qa.php.net/phpt_details.php#expectf_section :)

**Pull request merged**